hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterHeartbeatManager.java

```diff
@@ -518,45 +554,39 @@ private void pingMemberIfRequired(long now, MemberImpl member) {
         if (!icmpEnabled) {
```

Comment:
`if (!icmpEnabled || icmpParallelMode) return;` ?

Reply:
This is for the legacy mode, if the parallel mode is ON, we don't really need that. @metanet identified in a comment he send me.